File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length() ) return false;

        int[] m1=new int[26];
        int[] m2=new int[26];
        for(int i=0;i<s.length() ; i++){
            m1[s.charAt(i) - 'a']++;
            m2[t.charAt(i) - 'a']++;
        }
        if( Arrays.equals(m1,m2) ) return true;
        return false;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        if len(s) != len(t):
            return False
        f1 = {}
        for i in range(0, len(s)):
            f1[s[i]] = f1.get(s[i], 0) + 1
            f1[t[i]] = f1.get(t[i], 0) - 1
        for key, val in f1.items():
            if val != 0:
                return False
        return True
